api/collections/views.py

```diff
@@ -842,7 +841,7 @@ def get_object(self):
         return obj
 
 
-class CollectionLinkedRegistrationsRelationship(LinkedRegistrationsRelationship, CollectionMixin):
+class CollectionLinkedRegistrationsRelationship(CollectionLinkedNodesRelationship):
```

Comment:
We could more to DRY up some of these related views - `LinkedNodesRelationship` views and `LinkedRegistrationsRelationship` and similar. CollectionLinkedRegistrationsRelationship and CollectionLinkedNodesRelationship and CollectionLinkedPreprintsRelationship...

But this seems out of scope of this bugfix and would require more time to make sure we have proper test coverage before we do a wide sweep of refactoring.